src/constants.ts

```diff
@@ -1,8 +1,8 @@
-export const ADDON_ID = "my-addon";
+export const ADDON_ID = "@pyvignau/storybook-addon-figma-status-sync";
 export const TOOL_ID = `${ADDON_ID}/tool`;
 export const PANEL_ID = `${ADDON_ID}/panel`;
 export const TAB_ID = `${ADDON_ID}/tab`;
-export const KEY = `my-addon`;
+export const KEY = `storybook-addon-figma-status-sync`;
```

Comment:
You want to keep this one lightweight, it's the key that would be used in `addons.setConfig` or in `parameters` for your options. I'd use `figmaStatusSync`.

```suggestion
export const KEY = `figmaStatusSync`;
```